src/utils/setAuthToken.js

```diff
@@ -1,7 +1,12 @@
 import axios from 'axios';
 import config from 'Config';
+import Cookies from 'js-cookie';
 
 export default () => {
+  // Comment: if 'token' cookie is present and its not the same as localstorage token then set the new cookie token on localstorage
+  const tokenCookie = Cookies.get('token');
+  if (tokenCookie && Cookies.get('token') !== localStorage.token) localStorage.setItem('token', tokenCookie);
```

Comment:
I think that it is correct to overwrite nexchange token here. but we need to indicate somehow that we had the user before the affiliate in the future